{
  "id": "CVE-2020-24448",
  "assigner": "secure@intel.com",
  "Published": "2021-02-17T13:15:16.2Z",
  "Modified": "2021-02-23T13:45:13.053Z",
  "last-modified": "2021-02-23T13:45:13.053Z",
  "summary": "Uncaught exception in some Intel(R) Graphics Drivers before version 15.33.51.5146 may allow an authenticated user to potentially enable denial of service via local access.",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "LOCAL"
  },
  "impact": {
    "availability": "PARTIAL",
    "confidentiality": "NONE",
    "integrity": "NONE"
  },
  "cvss": 2.1,
  "exploitabilityScore": 3.9,
  "impactScore": 2.9,
  "cvss-vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P",
  "references": [
    "https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00438.html"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:intel:graphics_drivers:14.40:*:*:*:*:*:*:*",
    "cpe:2.3:a:intel:graphics_drivers:15.33:*:*:*:*:*:*:*",
    "cpe:2.3:a:intel:graphics_drivers:15.33.49.5100:*:*:*:*:*:*:*",
    "cpe:2.3:a:intel:graphics_drivers:15.33.50.5129:*:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:intel:graphics_drivers:14.40:*:*:*:*:*:*:*",
    "cpe:2.3:a:intel:graphics_drivers:15.33:*:*:*:*:*:*:*",
    "cpe:2.3:a:intel:graphics_drivers:15.33.49.5100:*:*:*:*:*:*:*",
    "cpe:2.3:a:intel:graphics_drivers:15.33.50.5129:*:*:*:*:*:*:*"
  ],
  "vendors": [
    "intel"
  ],
  "products": [
    "graphics_drivers"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:intel:graphics_drivers"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:intel:graphics_drivers"
  ],
  "cwe": "NVD-CWE-Other",
  "vulnerable_configuration_cpe_2_2": []
}